tracer/src/Datadog.FleetInstaller/FusionApi/Fusion+CreateAssemblyEnum.cs

```diff
@@ -0,0 +1,157 @@
+﻿// <copyright file="Fusion+CreateAssemblyEnum.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Based on https://github.com/JetBrains/msbuild/blob/5915c95cd0aeecfb9719549eb4b7c07b55a78a48/src/Tasks/NativeMethods.cs#L434
```

Comment:
JetBrains? Why not reference dotnet/msbuild? 😅

Reply:
riiiight 😅

Reply:
```suggestion
// Based on https://github.com/dotnet/msbuild/blob/5915c95cd0aeecfb9719549eb4b7c07b55a78a48/src/Tasks/NativeMethods.cs#L434
```